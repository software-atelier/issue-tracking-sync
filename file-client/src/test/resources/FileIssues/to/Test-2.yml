---
key: "Test-2"
summary: "First Issue on Disk"
status: "new"
description: "This issue will be updated"
lastUpdated: 2021-08-26T07:19
attributes: { }
comments: [ ]
attachments: [ ]
