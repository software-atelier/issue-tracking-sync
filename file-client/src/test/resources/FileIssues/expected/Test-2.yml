---
key: "Test-2"
summary: "Issue is updated"
status: "changed"
description: |
  This issue does already exists on target.
  After synchronization some fields are updated.
lastUpdated: 2021-08-27T12:07
attributes: { }
comments: [ ]
attachments: [ ]
